be/src/udf/udf.h

```diff
@@ -762,6 +766,45 @@ struct LargeIntVal : public AnyVal {
     }
 };
 
+struct HllVal : public StringVal {
+
```

Comment:
I think HllFullVal is a better name, because your HllVal only represent the Full type.